src/modules/username.rs

```diff
@@ -16,12 +16,18 @@ const USERNAME_ENV_VAR: &str = "USERNAME";
 ///     - The current user isn't the same as the one that is logged in (`$LOGNAME` != `$USER`) [2]
 ///     - The user is currently connected as an SSH session (`$SSH_CONNECTION`) [3]
 pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
-    let username = context.get_env(USERNAME_ENV_VAR)?;
+    let mut username = context.get_env(USERNAME_ENV_VAR)?;
 
     let mut module = context.new_module("username");
     let config: UsernameConfig = UsernameConfig::try_load(module.config);
 
     let is_root = is_root_user();
+
+    cfg!(target_os = "windows");
+    if is_root {
```

Comment:
The `cfg!`-macro is an expression that returns a boolean value.
```suggestion
    if cfg!(target_os = "windows") && is_root {
```

Reply:
Apologies for the rookie mistake. Fixed now